lib/transaction/transaction.js

```diff
@@ -205,6 +205,10 @@ Transaction.prototype._isInvalidSatoshis = function() {
 };
 
 Transaction.prototype._hasFeeError = function(opts) {
+  if (this._getUnspentValue() < 0) {
```

Comment:
What I'm suggesting is something more along the lines of: https://github.com/braydonf/bitcore/commit/9c55b02f4b6a81e48918466013e708a9877d5eb8